core/templates/pages/skill-editor-page/modal-templates/add-misconception-modal.controller.spec.ts

```diff
@@ -121,4 +121,24 @@ describe('Add Misconception Modal Controller', function() {
     $scope.saveMisconception();
     expect($uibModalInstance.close).not.toHaveBeenCalled();
   });
+
+  // eslint-disable-next-line max-len
+  it('should check if the content length of misconception notes is within 1000 chars',
+    function() {
+      $scope.misconceptionNotes = 'a'.repeat(1000);
+      expect($scope.isMisconceptionNotesHtmlLengthExceeded()).toBe(false);
+
+      $scope.misconceptionNotes = 'a'.repeat(1001);
+      expect($scope.isMisconceptionNotesHtmlLengthExceeded()).toBe(true);
+    });
+
+  // eslint-disable-next-line max-len
+  it('should check if the content length of misconception feedback is within 1000 chars',
+    function() {
+      $scope.misconceptionFeedback = 'a'.repeat(1000);
+      expect($scope.isMisconceptionFeedbackHtmlLengthExceeded()).toBe(false);
```

Comment:
I'm a bit confused... The HTML is not something the user would input directly... Putting limits on the HTML size could be confusing to the user. How do we communicate this limit to the user (on the UI)?

Reply:
@nithusha21 I believe you have missed some threads where we had discussion regarding the same . We came to a conclusion to limit the total html chars rather than the content inside of that html tags . It also fullproof against things like image tags where actual content of an image can break the limit instantly ( image data can contain thousands of character ) . You can go through the top threads to read about it . Thanks !!

Reply:
I did go through the threads, and I am not convinced this is the right thing to do. Left a more detailed comment in the other thread, we can continue the discussion there.